gateway-service-impl/src/main/java/org/hypertrace/gateway/service/explore/entity/EntityRequestHandler.java

```diff
@@ -76,21 +73,15 @@ public ExploreResponse.Builder handleRequest(
 
     builder.addAllRow(
         entityServiceEntityFetcher.getResults(requestContext, exploreRequest, entityIds));
-
-    // If there's a Group By in the request, we need to do the sorting and pagination ourselves.
```

Comment:
Any reason why we have removed this?

Reply:
I checked all the scenarios i dont think this is needed.